lib/fog/compute/google/models/firewall.rb

```diff
@@ -28,6 +28,15 @@ def save
           reload
         end
 
+        def update
```

Comment:
So instead of calling update, could we make this be logic in `save`? So save will insert if it doesn't exist or update if it does?